icu4c/source/i18n/unicode/nounit.h

```diff
@@ -32,77 +32,43 @@ U_NAMESPACE_BEGIN
  * @see NumberFormatter
  * @draft ICU 60
  */
-class U_I18N_API NoUnit: public MeasureUnit {
-public:
+namespace NoUnit {
```

Comment:
lgtm but enough of a change to warrant an API proposal

Reply:
I sent the API proposal this morning.